rllib/utils/nested_dict.py

```diff
@@ -181,7 +181,12 @@ def get(
     def __getitem__(self, k: SeqStrType) -> T:
         output = self.get(k)
         if isinstance(output, NestedDict):
-            raise IndexError("Use get for partial indexing.")
+            raise IndexError(
+                f"Key `{k}` is not a complete key in the given "
+                f"{self.__class__.__name__}. It results in a container "
+                f"with subkeys {set(output.keys())}. To get partial indexing, "
```

Comment:
-> "To use partial indexing and thus retrieve a sub-structure ..."

Reply:
I removed partial indexing error all together due the valid comment above. Also update the examples in the docstring  to show that index error is not raised anymore.